src/modules/submissions.js

```diff
@@ -706,6 +734,30 @@
 		}
 	} );
 
+	function getDraftArticleSubcats( cat ) {
+		var articleSubcats = [],
+			// This is so a new version of AFCH will invalidate the articleSubcats cache
+			lsKey = 'afch-' + AFCH.consts.version + 'draft-article-subcats';
+		if ( window.localStorage && window.localStorage[ lsKey ] ) {
+			articleSubcats = JSON.parse( window.localStorage[ lsKey ] );
+			return articleSubcats;
+		} else {
+			return getDraftArticleSubcatsInner( cat ).then( function ( articleSubcats ) { // there has to be a return here so that the function will return something
+				// If possible, cache the articleSubcats data!
+				if ( window.localStorage ) {
+					try {
+						window.localStorage.removeItem( lsKey );
```

Comment:
this line was only necessary during testing, so it can be removed now

Reply:
Are you talkin about ``window.localStorage.removeItem( lsKey );`` ?